utils/build/build.js

```diff
@@ -283,7 +283,7 @@ for (const webPackage of ['html-reporter', 'recorder', 'trace-viewer']) {
       'vite',
       'build',
       ...(watchMode ? ['--watch', '--minify=false'] : []),
-      ...(withSourceMaps ? ['--sourcemap'] : []),
+      ...(withSourceMaps ? ['--sourcemap=inline'] : []),
```

Comment:
Note: This is a drive-by that because of a recent DevTools regression which prevents source-maps from working correctly with single token domains.